### PR TITLE
French translation for `accent-color` CSS property reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR aims to create a French translation for `accent-color` CSS property reference.

### CODE REVIEW

File: files/fr/web/css/accent-color/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: accent-color
+slug: Web/CSS/accent-color
+browser-compat: css.properties.accent-color
+translation_of: Web/CSS/accent-color
+---
+<div>{{CSSRef}}{{SeeCompatTable}}</div>
+
+<p>La propriété <a href="/fr/docs/Web/CSS">CSS</a> <strong><code>accent-color</code></strong> définit la couleur d'accentuation ({{Glossary("accent")}}) des éléments. Une accentuation est présent sur des éléments HTML tels que des {{HTMLElement("input")}} de type <code><a href="/fr/docs/Web/HTML/Element/input/checkbox">checkbox</a></code> ou <code><a href="/fr/docs/Web/HTML/Element/input/radio">radio</a></code>.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/accent-color/index.html:9]: "accentuation est présent" > "accentuation est présente"
(thx Grammalecte ^^ )

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Thanks for first fixes already. ~I just~ Grammalecte spotted a typo.

File: files/fr/web/css/accent-color/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/accent-color/index.html]: Bien vu !

**Review by SphinxKnight — APPROVED** [review]
👍

**Pull request merged**